Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -738,10 +751,19 @@ DayOfWeek: !dictionary
     vrijdag: 5
     zaterdag: 6
     zondag: 0
+    maandagen: 1
```

Comment:
@tellarin is this necessary? Or can this be normalized in code to use the day without the `en` suffix?

Reply:
@iMicknl, the regexes WeekDayRegex and SingleWeekDayRegex can be refactored so that the suffixed variations become unnecessary. (we proceeded in this way following the work done in Turkish where DayOfWeek also contains various suffixed entries). 
Should we update the file?

Reply:
@LionbridgeCSII, I'll merge this PR as-is, but please make the suggested change in a follow up PR.